Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -242,15 +273,15 @@ void evdevDevice::UpdateInput()
   // Run through all evdev events
   // libevdev will keep track of the actual controller state internally which can be queried
   // later with libevdev_fetch_event_value()
-  input_event ev;
   int rc = LIBEVDEV_READ_STATUS_SUCCESS;
-  do
+  while (rc >= 0)
   {
-    if (rc == LIBEVDEV_READ_STATUS_SYNC)
+    input_event ev;
+    if (LIBEVDEV_READ_STATUS_SYNC == rc)
```

Comment:
Is there any reason this was inverted? Seems inconsistent with the rest of the code.

Reply:
No particular reason.